### PR TITLE
PMM-9384 first part

### PR DESCRIPTION
**Pull request by vyarosh**
Implemented PMM-T1106, PMM-T486 and PMM-T1104

### CODE REVIEW

**Comment by vyarosh** [comment]
https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-tests/detail/pmm2-ui-tests/11411/pipeline

**Comment by vyarosh** [comment]
https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-test-nightly/detail/pmm2-ui-test-nightly/909/pipeline

File: tests/pages/pmmSettingsPage.js [hunk 1]
```diff
@@ -90,73 +90,93 @@ module.exports = {
   },
   tooltips: {
     dbaas: {
+      iconLocator: locate('$advanced-dbaas').find('div[class$="-Icon"]'),
       text: 'Option to enable/disable DBaaS features. Disabling DBaaS does not suspend or remove running clusters.',
       link: links.dbaasDocs,
     },
     stt: {
+      iconLocator: '//div[@data-testid="advanced-stt"]//div/div/div/div',
```
Comment (puneet0191) [inline tests/pages/pmmSettingsPage.js:98]: any reason why not using locate here?

File: tests/verifyPMMSettingsPageFunctionality_test.js [hunk 2]
```diff
@@ -148,18 +144,11 @@ Scenario(
     // Verify tooltips for Communication > Email fields
     for (const o of Object.keys(pmmSettingsPage.communication.email)) {
```
Comment (puneet0191) [inline tests/verifyPMMSettingsPageFunctionality_test.js:145]: I'd rather use better naming for variable, not o, use 
`const field of Object.keys..`

File: tests/pages/pmmSettingsPage.js [hunk 3]
```diff
@@ -180,7 +200,7 @@ module.exports = {
     alertmanagerRulesLabel: '$alertmanager-rules-label',
     alertmanagerButton: '$alertmanager-button',
     amUrlLabel: locateLabel('form-field-am-url'),
-    applyButton: '//button[@type="submit"]',
```
Comment (puneet0191) [inline tests/pages/pmmSettingsPage.js:183]: Your tests fail on Nightly exactly for the same reason, https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-test-nightly/detail/pmm2-ui-test-nightly/909/tests

**Review by puneet0191 — CHANGES_REQUESTED** [review]

File: tests/pages/pmmSettingsPage.js (hunk 1 above)
Comment (vyarosh) [reply tests/pages/pmmSettingsPage.js:98]: I've moved this locator from: https://github.com/percona/pmm-ui-tests/blob/main/tests/pages/pmmSettingsPage.js#L218

Would you like me to update it using `locate()` ?

File: tests/verifyPMMSettingsPageFunctionality_test.js (hunk 2 above)
Comment (vyarosh) [reply tests/verifyPMMSettingsPageFunctionality_test.js]: refactored

File: tests/pages/pmmSettingsPage.js (hunk 3 above)
Comment (vyarosh) [reply tests/pages/pmmSettingsPage.js:183]: fixed, I've missed that I've changed it to css selector

**Comment by vyarosh** [comment]
fixed 2 issues:
https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-tests/detail/pmm2-ui-tests/11419/pipeline
https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-test-nightly/detail/pmm2-ui-test-nightly/912/pipeline

**Review by puneet0191 — APPROVED** [review]

**Pull request merged**